viewer/script/copyPackage.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+function copyPackage(version) {
+  const jsonString = fs.readFileSync('./package.json', 'utf8')
+  const json = JSON.parse(jsonString);
+
+  if(version) {
+    // anything else we should remove?
+    json.version = version;
+    delete json.private;
+    delete json.scripts.publishscript;
+    delete json.scripts.prepublishscript;
```

Comment:
Just so I know, will the package be public if I use this script?

Reply:
I don't think so. We already do npm publish without the private parameter in the json file.